### ISSUE TITLE
Inconsistency in species defined in GEOS-Chem and HEMCO config following pull request #2318

### ISSUE THREAD

**Issue by msulprizio** [geoschem/geos-chem#2459]
### Your name

Melissa Sulprizio

### Your affiliation

Harvard

### What happened? What did you expect to happen?

The current dev/14.5.0 branch (tagged 14.5.0-alpha.11) lists species ALK6 in HEMCO_Config.rc but there is no such species defined in the species database nor geoschem_config.yml. Commit https://github.com/geoschem/geos-chem/commit/7cd5aefea560f18c80280ad4edc5d26d4786120c (added atop pull request #2318) adding CEDS emissions for new species added in #2318. In particular, these lines were added to HEMCO_Config.rc for new species from CEDS:

```
0 CEDS_C6H14_AGR  $ROOT/CEDS/v2021-06/$YYYY/ALK4_hexanes-em-anthro_CMIP_CEDS_$YYYY.nc  ALK4_hexanes_agr  1750-2019/1-12/1/0 C xy   kg/m2/s ALK6  2401                1 5
0 CEDS_C6H14_ENE  $ROOT/CEDS/v2021-06/$YYYY/ALK4_hexanes-em-anthro_CMIP_CEDS_$YYYY.nc  ALK4_hexanes_ene  1750-2019/1-12/1/0 C xyL* kg/m2/s ALK6  2406/706/315        1 5
0 CEDS_C6H14_IND  $ROOT/CEDS/v2021-06/$YYYY/ALK4_hexanes-em-anthro_CMIP_CEDS_$YYYY.nc  ALK4_hexanes_ind  1750-2019/1-12/1/0 C xyL* kg/m2/s ALK6  2407/707/316        1 5
0 CEDS_C6H14_TRA  $ROOT/CEDS/v2021-06/$YYYY/ALK4_hexanes-em-anthro_CMIP_CEDS_$YYYY.nc  ALK4_hexanes_tra  1750-2019/1-12/1/0 C xy   kg/m2/s ALK6  2411/711            1 5
0 CEDS_C6H14_RCO  $ROOT/CEDS/v2021-06/$YYYY/ALK4_hexanes-em-anthro_CMIP_CEDS_$YYYY.nc  ALK4_hexanes_rco  1750-2019/1-12/1/0 C xy   kg/m2/s ALK6  2409/709            1 5
0 CEDS_C6H14_SLV  $ROOT/CEDS/v2021-06/$YYYY/ALK4_hexanes-em-anthro_CMIP_CEDS_$YYYY.nc  ALK4_hexanes_slv  1750-2019/1-12/1/0 C xy   kg/m2/s ALK6  2407/707            1 5
0 CEDS_C6H14_WST  $ROOT/CEDS/v2021-06/$YYYY/ALK4_hexanes-em-anthro_CMIP_CEDS_$YYYY.nc  ALK4_hexanes_wst  1750-2019/1-12/1/0 C xy   kg/m2/s ALK6  26                  1 5

0 CEDS_TMB_AGR    $ROOT/CEDS/v2021-06/$YYYY/TMB-em-anthro_CMIP_CEDS_$YYYY.nc           TMB_agr           1750-2019/1-12/1/0 C xy   kg/m2/s HCOOH 26                  1 5
0 CEDS_TMB_ENE    $ROOT/CEDS/v2021-06/$YYYY/TMB-em-anthro_CMIP_CEDS_$YYYY.nc           TMB_ene           1750-2019/1-12/1/0 C xyL* kg/m2/s HCOOH 26/315              1 5
0 CEDS_TMB_IND    $ROOT/CEDS/v2021-06/$YYYY/TMB-em-anthro_CMIP_CEDS_$YYYY.nc           TMB_ind           1750-2019/1-12/1/0 C xyL* kg/m2/s HCOOH 26/316              1 5
0 CEDS_TMB_TRA    $ROOT/CEDS/v2021-06/$YYYY/TMB-em-anthro_CMIP_CEDS_$YYYY.nc           TMB_tra           1750-2019/1-12/1/0 C xy   kg/m2/s HCOOH 26                  1 5
0 CEDS_TMB_RCO    $ROOT/CEDS/v2021-06/$YYYY/TMB-em-anthro_CMIP_CEDS_$YYYY.nc           TMB_rco           1750-2019/1-12/1/0 C xy   kg/m2/s HCOOH 26                  1 5
0 CEDS_TMB_SLV    $ROOT/CEDS/v2021-06/$YYYY/TMB-em-anthro_CMIP_CEDS_$YYYY.nc           TMB_slv           1750-2019/1-12/1/0 C xy   kg/m2/s HCOOH 26                  1 5
0 CEDS_TMB_WST    $ROOT/CEDS/v2021-06/$YYYY/TMB-em-anthro_CMIP_CEDS_$YYYY.nc           TMB_wst           1750-2019/1-12/1/0 C xy   kg/m2/s HCOOH 26                  1 5

0 CEDS_OTH_AGR    $ROOT/CEDS/v2021-06/$YYYY/OTHER_VOC-em-anthro_CMIP_CEDS_$YYYY.nc     OTHER_VOC_agr     1750-2019/1-12/1/0 C xy   kg/m2/s ALK6 26                   1 5
0 CEDS_OTH_ENE    $ROOT/CEDS/v2021-06/$YYYY/OTHER_VOC-em-anthro_CMIP_CEDS_$YYYY.nc     OTHER_VOC_ene     1750-2019/1-12/1/0 C xyL* kg/m2/s ALK6 26/315               1 5
0 CEDS_OTH_IND    $ROOT/CEDS/v2021-06/$YYYY/OTHER_VOC-em-anthro_CMIP_CEDS_$YYYY.nc     OTHER_VOC_ind     1750-2019/1-12/1/0 C xyL* kg/m2/s ALK6 26/316               1 5
0 CEDS_OTH_TRA    $ROOT/CEDS/v2021-06/$YYYY/OTHER_VOC-em-anthro_CMIP_CEDS_$YYYY.nc     OTHER_VOC_tra     1750-2019/1-12/1/0 C xy   kg/m2/s ALK6 26                   1 5
0 CEDS_OTH_RCO    $ROOT/CEDS/v2021-06/$YYYY/OTHER_VOC-em-anthro_CMIP_CEDS_$YYYY.nc     OTHER_VOC_rco     1750-2019/1-12/1/0 C xy   kg/m2/s ALK6 26                   1 5
0 CEDS_OTH_SLV    $ROOT/CEDS/v2021-06/$YYYY/OTHER_VOC-em-anthro_CMIP_CEDS_$YYYY.nc     OTHER_VOC_slv     1750-2019/1-12/1/0 C xy   kg/m2/s ALK6 26                   1 5
0 CEDS_OTH_WST    $ROOT/CEDS/v2021-06/$YYYY/OTHER_VOC-em-anthro_CMIP_CEDS_$YYYY.nc     OTHER_VOC_wst     1750-2019/1-12/1/0 C xy   kg/m2/s ALK6 26                   1 5
```

@kelvinb @ktravis213 Can you confirm which species these emissions should be applied to? ALK7? STYR?

https://github.com/geoschem/geos-chem/blob/a5fba9c88ad279095026be17c31a40bb84802eda/run/shared/species_database.yml#L213-L218

https://github.com/geoschem/geos-chem/blob/a5fba9c88ad279095026be17c31a40bb84802eda/run/shared/species_database.yml#L5271-L5276

### What are the steps to reproduce the bug?

View run directory config files in the dev/14.5.0 branch:
https://github.com/geoschem/geos-chem/tree/dev/14.5.0/run

### What GEOS-Chem version were you using?

14.5.0-alpha.11

### What environment were you running GEOS-Chem on?

Local cluster

### What compiler and version were you using?

N/A

### Will you be addressing this bug yourself?

Yes, but I will need some help

### In what configuration were you running GEOS-Chem?

GCClassic

### What simulation were you running?

Full chemistry

### As what resolution were you running GEOS-Chem?

4x5

### What meterology fields did you use?

GEOS-FP

**Comment by ktravis213** [comment]
Alk6 and Alk7 are the same. I some point I called it ALK7 and switched to ALk6 so maybe there was some confusion with the emissions pull request. So whichever is used in fullchem.eqn should be used throughout. I will check shortly but am a little slow as I am attending a conference.

**Comment by msulprizio** [comment]
Thanks @ktravis213. It should be ALK7 in that case then since that is what is currently used in fullchem.eqn.

The only remaining question I had was whether CEDS "other VOCs" should go to styrene (STYR)? @djxjacob was under the impression that styrene should have an anthropogenic emissions source.

**Comment by ktravis213** [comment]
Here is @kelvinhb advice on styrene. 
"Styrene doesn't have its own category in CEDS and the CEDS inventory isn't clear at all on how to divide up their "other aromatics" category, so we just have it turned off. Lacking any guidance on how to allocate the other aromatics, I definitely don't want to send them all to styrene, because it's way more reactive than basically any other aromatic. The "other VOCs" category, though, is pretty clearly larger alkanes, so it makes sense for that to go to ALK7. There's also CEDS categories for larger alkenes (which may include styrene as well? unclear), ethers, and esters that we have turned off. In my AEROMMA analysis I'm working on at least testing what it would mean to turn those on and speciate them, so that in future we'll at least have a starting point to play with them. But for now I don't think we can justify turning them on."  So our recommendation is to leave the speciation for styrene from CEDS as a research question.  It can certainly be turned on for regional inventories more easily.

**Comment by msulprizio** [comment]
Thanks @ktravis213 and @kelvinhb!

**Comment by msulprizio** [comment]
This was addressed by https://github.com/geoschem/geos-chem/pull/2462 which has now been merged into the `dev/14.5.0` branch.

**Comment by msulprizio** [comment]
Based on the discussion above we decided on using ALK7. However, after further discussion at the September 2024 GEOS-Chem Steering Committee meeting and with @ktravis213 and @kelvinhb, we should use ALK6. This is also consistent with the naming in [Travis, K.T., et al. EGUsphere (2024)](https://egusphere.copernicus.org/preprints/2024/egusphere-2024-951/)
